spanner/api/Spanner.Samples/IdentifierCaseSensitivityAsyncPostgre.cs

```diff
@@ -0,0 +1,127 @@
+
+        // PostgreSQL case sensitivity with mutations.
+        // Mutations: Column names in mutations are always case-insensitive, regardless whether the
+        // columns were double-quoted or not during creation.
+        using var cmd = connection.CreateInsertCommand("Concerts", new SpannerParameterCollection
+                {
+                        { "ConcertId", SpannerDbType.Int64, 1 },
+                        { "Location", SpannerDbType.String, "Venue 1" },
+                        { "Time", SpannerDbType.Timestamp, DateTime.UtcNow }
+                });
+        await cmd.ExecuteNonQueryAsync();
+
+        // PostgreSQL case sensitivity with queries.
+        var selectCommand = connection.CreateSelectCommand("SELECT * FROM Concerts");
+        using var reader = await selectCommand.ExecuteReaderAsync();
+        while (await reader.ReadAsync())
+        {
+            Console.WriteLine("ConcertId : "
+            // ConcertId was not double quoted while table creation, so it is automatically folded to lower case.
+            // Accessing the column by its name in a result set must therefore use all lower-case letters.
+            + reader.GetFieldValue<long>("concertid")
+            // Location and Time were double - quoted during creation,
+            // and retain their mixed case when returned in a result set.
+            + " Location : "
+            + reader.GetFieldValue<string>("Location")
+            + " Time : "
+            + reader.GetFieldValue<DateTime>("Time"));
+        }
+
+        // PostgreSQL case sensitivity with aliases.
+        // Aliases : They are also identifiers, and specifying an alias in double quotes will make the alias retain its case.
+        var selectAliasCommand = connection.CreateSelectCommand("SELECT concertid AS \"ConcertId\", \"Location\" AS \"venue\", \"Time\" FROM Concerts");
+        using var dataReader = await selectAliasCommand.ExecuteReaderAsync();
+        while (await dataReader.ReadAsync())
+        {
+            // The aliases are double-quoted and therefore retain their mixed case.
+            Console.WriteLine("ConcertId : "
+            + dataReader.GetFieldValue<long>("ConcertId")
+            + " Location : "
+            + dataReader.GetFieldValue<string>("venue")
+            + " Time : "
+            + dataReader.GetFieldValue<DateTime>("Time"));
+        }
+
+        // PostgreSQL case sensitivity with DML statements.
+        // DML statements must also follow the PostgreSQL case rules.
+        var dmlCommand = connection.CreateDmlCommand("INSERT INTO Concerts (ConcertId, \"Location\", \"Time\") VALUES($1, $2, $3)",
+            new SpannerParameterCollection
+            {
+                { "p1", SpannerDbType.Int64, 2 },
+                { "p2", SpannerDbType.String, "Venue 2" },
+                { "p3", SpannerDbType.Timestamp, DateTime.UtcNow }
+            });
+
+        var count = await dmlCommand.ExecuteNonQueryAsync();
```

Comment:
This is one of the main problems with having samples that do many things at once. We are barely testing this sample.